googleapiclient/model.py

```diff
@@ -140,11 +142,7 @@ def request(self, headers, path_params, query_params, body_value):
     query = self._build_query(query_params)
     headers['accept'] = self.accept
     headers['accept-encoding'] = 'gzip, deflate'
-    if 'user-agent' in headers:
-      headers['user-agent'] += ' '
-    else:
-      headers['user-agent'] = ''
-    headers['user-agent'] += 'google-api-python-client/%s (gzip)' % __version__
+    headers['x-goog-api-client'] = 'gdcl/%s gl-python/%s' % (__version__, _PY_VERSION)
```

Comment:
Is it alright to override whatever was in `x-goog-api-client`?

Reply:
I was about to comment on that. Do we have python veneers that run atop this client? I'd expect they might set `gccl/` and pass the header down here, which might append `gdcl/ and gl-python/`. It's probably better to preserve and append to any existing data.